lib/served/resource/json_api_resource.rb

```diff
@@ -0,0 +1,136 @@
+# frozen_string_literal: true
+
+module Served
+  module Resource
+    # Resources that implement JSONApi Spec should use this class
+    class JsonApiResource < Served::Resource::Base
```

Comment:
why do you favour a module over its own class? I see it more like an adapter and I choose which adapter I want to use?